packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_set_default_loan_scheme.test.ts

```diff
@@ -0,0 +1,150 @@
+import { DeFiDRpcError, GenesisKeys } from '@defichain/testcontainers'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder } from '../../src'
+import { fundEllipticPair, sendTransaction } from '../test.utils'
+import { WIF } from '@defichain/jellyfish-crypto'
+import BigNumber from 'bignumber.js'
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import { Testing } from '@defichain/jellyfish-testing'
+
+const container = new LoanMasterNodeRegTestContainer()
+const testing = Testing.create(container)
+
+let providers: MockProviders
+let builder: P2WPKHTransactionBuilder
+
+beforeAll(async () => {
+  await testing.container.start()
+  await testing.container.waitForWalletCoinbaseMaturity()
+
+  providers = await getProviders(testing.container)
+  providers.setEllipticPair(WIF.asEllipticPair(GenesisKeys[GenesisKeys.length - 1].owner.privKey))
```

Comment:
Why `GenesisKeys[GenesisKeys.length - 1]` ? What happens with `GenesisKeys[0]`, the default one ?

Reply:
Good question. I copied this from other code. 
Just tested with 0. And all the test units were unable to run successfully.
FYI Genesis.length = 8
Maybe Fuxing can answer your question.

Reply:
Just tested and it throws error 
` "DeFiDRpcError: 'DefaultLoanSchemeTx: tx not from foundation member! (code 16)', code: -26"`.
Might be good to add a test with another container set to default Genesis[0] to explicit this requirement

Reply:
The comment in testkeys.ts is incorrect `* 2 first and 2 last of genesis MNs acts as foundation members`
 From my testing, 2 last are belongs to foundation member, not 2 first

Reply:
Yes I see that as well. 
I can't understand why the first two addresses don't pass the `HasFoundationAuth()` on `ain` side (https://github.com/DeFiCh/ain/blob/ff53dcee23db2ffe0da9b147a0a53956f4e7ee31/src/masternodes/mn_checks.cpp#L429).
Any ideas @fuxingloh ?

Reply:
Current limitations on jellyfish side. 

Currently `jellyfish-transaction-*` can only sign transactions using bech32 addresses (segwit). The last 2 addresses use bech32, they start with bcrt.

This ticket will fix it but heavily deprioritized as within jellyfish ecosystem we only need bech32 for now https://github.com/DeFiCh/jellyfish/issues/384